atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -293,13 +293,22 @@ static String getFilteredStackTrace(String filter) {
     }
 
     static Column createColumn(Cell cell, Value value) {
+        return createColumn(cell, value, value.getTimestamp());
+    }
+
+    static Column createColumn(Cell cell, Value value, long writeTimestamp) {
         byte[] contents = value.getContents();
-        long timestamp = value.getTimestamp();
-        ByteBuffer colName = makeCompositeBuffer(cell.getColumnName(), timestamp);
+        ByteBuffer colName = makeCompositeBuffer(cell.getColumnName(), value.getTimestamp());
         Column col = new Column();
         col.setName(colName);
         col.setValue(contents);
-        col.setTimestamp(timestamp);
+        col.setTimestamp(writeTimestamp);
+        return col;
+    }
+
+    static Column createColumnAndOverwriteTimestamp(Cell cell, Value value) {
```

Comment:
This provides identical functionality to `createColumn(Cell, Value)`. Can we `@Deprecate` the original method?